server/auth/auth.go

```diff
@@ -415,6 +424,15 @@ func (m machine) Type() string {
 	return "machine"
 }
 
+func (m machine) Raw() []knox.RawPrincipal {
+	return []knox.RawPrincipal{
+		{
+			ID:   string(m),
+			Type: "machine",
```

Comment:
Call `u.Type()` here, rather than repeating the string. That way there is a single source of truth for the value of that string.